openapi/saps4hana.itcm.accountreceivable/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["sap", "saps4hana", "accountreceivable", "paid"]
```

Comment:
You can add a space for key words
"account receivable"

Reply:
Fixed this